### ISSUE TITLE
Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by sandeepsinghs** [mmazzarolo/react-native-modal-datetime-picker#207]
### When I try to run the example project on Android  then I am getting
![datetimepicker_error](https://user-images.githubusercontent.com/14754191/47785285-44f5e580-dd2e-11e8-8fbe-5b1b4e3f6174.png)
 this issue.

The example project URL as [https://github.com/mmazzarolo/react-native-modal-datetime-picker/tree/master/example](url) 

Because of this I am not able to run project.

**Comment by mmazzarolo** [comment]
Could you try removing the `react-native-modal-datetime-picker` to see if it works?

**Comment by mmazzarolo** [comment]
> I am trying to run yours example project. If I will remove react-native-modal-datetime-picker from project then it is a blank project,  

But do you get the error anyway?
I just tried the example and it is working fine for me so I'm not really sure it's related to the library. 
Did you already check in the react-native issues?
